chainer/links/model/vision/vgg.py

```diff
@@ -168,18 +164,19 @@ def forward(self, x, layers=None, **kwargs):
         .. warning::
 
            ``test`` argument is not supported anymore since v2.
-           Instead, use ``chainer.using_config('train', train)``.
+           Instead, use ``chainer.using_config('train', False)``.
```

Comment:
For clarity, could you add `to run in test mode` at the end of the sentence?

Reply:
OK